src/main/java/com/vamshi/springboot/mytodolist/entity/TodoList.java

```diff
@@ -0,0 +1,87 @@
+package com.vamshi.springboot.mytodolist.entity;
+
+import org.springframework.beans.propertyeditors.StringTrimmerEditor;
+import org.springframework.web.bind.WebDataBinder;
+import org.springframework.web.bind.annotation.InitBinder;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Null;
+
+@Entity
+@Table(name = "todolist")
+public class TodoList {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    public int id;
+
+    @NotNull(message="is required")
+    @Column(name = "description")
+    public String description;
+
+
+    @Column(name = "category", insertable=false)
+//    @Column(name = "category")
+    public String category;
+
+
+    @Column(name = "status", insertable=false)
+    public String status="todo";
+
+    @Column(name = "date")
+    public String date;
+
+
+    public int getId() {
```

Comment:
Why there are setters an getters instead of using Lombok?

Reply:
resolved